gnomad_qc/v5/data_ingestion/create_vds.py

```diff
@@ -0,0 +1,66 @@
+"""Script to combine gVCFs into a single VDS."""
+
+import argparse
+import logging
+
+import hail as hl
+from gnomad.utils.file_utils import create_vds
+
+from gnomad_qc.v5.resources.basics import dragen_tgp_vds
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("create_vds")
+logger.setLevel(logging.INFO)
+
+
+def main(args):
+    """Create VDS from table of GVCFs paths and write to output path."""
+    hl.init(
+        log="/tmp/gvcf_combiner.log", default_reference="GRCh38", tmp_dir=args.temp_path
+    )
+    gvcfs = args.gvcfs
+    temp_path = args.temp_path
+    save_path = args.temp_path + "combiner_plan.json"
+    output_path = args.output_path if args.output_path else dragen_tgp_vds.path
+
+    vds = create_vds(
+        gvcfs=gvcfs,
+        output_path=output_path,
+        temp_path=temp_path,
+        save_path=save_path,
+        use_genome_default_intervals=args.use_genome_default_intervals,
+        intervals=args.intervals,
+        gvcf_batch_size=args.gvcf_batch_size,
+    )
+    logger.info("VDS variant data schema: %s", vds.variant_data.describe())
+    logger.info("VDS reference data schema: %s", vds.reference_data.describe())
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--gvcfs", help="Path to file with only gVCF paths", required=True
+    )
+    parser.add_argument(
+        "--output-path", help="Path to write output VDS to", required=True
```

Comment:
I include the argument for testing flexibility. If it isnt provided, it will default to the dragen_tgp_vds resource path. We can iterate on this once we need the code to do more.